crypto/rsa/rsa_gen.c

```diff
@@ -57,13 +58,16 @@ int RSA_generate_multi_prime_key(RSA *rsa, int bits, int primes,
         else
             return 0;
     }
-
+#endif /* FIPS_MODE */
     return rsa_builtin_keygen(rsa, bits, primes, e_value, cb);
```

Comment:
Doesn't this end up ignoring `primes` in the FIPS case? Shouldn't it be an error if `primes != 2`

Reply:
I added your suggestion.